services/app-web/src/common-code/proto/healthPlanFormDataProto/toDomain.ts

```diff
@@ -441,12 +443,12 @@ const toDomain = (
         rateDateEnd: protoDateToDomain(rateInfo?.rateDateEnd),
         rateDateCertified: protoDateToDomain(rateInfo?.rateDateCertified),
         rateProgramIDs: rateInfo?.rateProgramIds ?? [],
-        actuaryCommunicationPreference: enumToDomain(
+        addtlActuaryCommunicationPreference: enumToDomain(
             mcreviewproto.ActuaryCommunicationType,
-            rateInfo?.actuaryCommunicationPreference
+            addtlActuaryCommunicationPreference
         ),
         stateContacts: cleanedStateContacts,
-        actuaryContacts: parseActuaryContacts(rateInfo),
+        addtlActuaryContacts: parseActuaryContacts(addtlActuaryContacts),
```

Comment:
🔍 This changes here looks risky to me. If we were to merge without the other related tickets I think we would cause a regression, because we discard the `rateInfo.[ACTUARY_FIELD]`  from the domain model data all together. 

However, those same fields are where old submissions are storing actuary contacts.

To make this change more able to merge by independently (without also requiring https://qmacbis.atlassian.net/browse/MR-2233) I believe we should still be reading from the rateInfos as a fallback. 

Something like 
        addtlActuaryContacts: parseActuaryContacts(addtlActuaryContacts) ?? parseActuaryContacts(rateInfo)

Reply:
resolved with [9bdeae2](https://github.com/CMSgov/managed-care-review/pull/1276/commits/9bdeae2339ccbad98f2a0c16d117a2274cb7c753)